dashboard/app/assets/stylesheets/application.scss

```diff
@@ -218,19 +218,19 @@ img.video_thumbnail {
     text-overflow: ellipsis;
   }
 
-  @media screen and (min-width: 971px) {
+  @media screen and (min-width: 1011px) {
```

Comment:
As it happens, 971px is the standard width that pegasus uses for "desktop", and the dashboard was using the same value because we try to keep the headers consistent between the two systems, despite the separate implementations.

Unfortunately, the Incubator link exceeds the available space, and so we have to now adjust the cutoffs for what's shown in the header.

Reply:
I've been using these breakpoints for all of the Pegasus rebrand pages: [shared/css/breakpoints.scss](https://github.com/code-dot-org/code-dot-org/blob/d2dffe78c33a1c969547fcb6fea0ace61726783d/shared/css/breakpoints.scss)